org.eclipse.nebula.widgets.nattable.examples.e4.rcp.feature/feature.xml

```diff
@@ -25,14 +25,14 @@
    </requires>
 
    <plugin
-         id="javax.annotation"
+         id="jakarta.annotation-api"
          download-size="28"
          install-size="58"
          version="0.0.0"
          unpack="false"/>
 
    <plugin
-         id="javax.inject"
+         id="jakarta.inject.jakarta.inject-api"
```

Comment:
Do those bundles already exist in the 2020-06 release?

Reply:
Why does it need to exist in a specific release?  It's available in Orbit.

I think it's actually better if we remove this entirely.  See comments below.